pkg/kubectl/describe.go

```diff
@@ -766,8 +766,8 @@ func (d *PersistentVolumeClaimDescriber) Describe(namespace, name string) (strin
 	})
 }
 
-// DescribeContainers is exported for consumers in other API groups that have container templates
-func DescribeContainers(containers []api.Container, containerStatuses []api.ContainerStatus, resolverFn EnvVarResolverFunc, out io.Writer) {
+// describeContainers is exported for consumers in other API groups that have container templates
+func describeContainers(containers []api.Container, containerStatuses []api.ContainerStatus, resolverFn EnvVarResolverFunc, out io.Writer) {
```

Comment:
@AdoHe yes now we are going to use it via the podtemplate describer so I think it's fine